src/app/collection/pages/collection-ncyte/footer/footer.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '~_vars.scss';
+
+.footer{
+    background-color: $dark-grey;
+    color: #fff;
+    min-height: 40vh;
+    min-width: 30vw;
+    margin-top: 50px;
+    padding: 100px 0px;
```

Comment:
You want to put a little more padding on the right and left.

Reply:
I don't know if you know this @devky690 but for margin and padding the notation is top right bottom left, so for padding you can do padding: 0 10px 0 10px; for example